controller/src/beerocks/master/tasks/association_handling_task.cpp

```diff
@@ -29,6 +29,8 @@ using namespace son;
 #define BEACON_MEASURE_MAX_ATTEMPTS 3
 #define REQUEST_RSSI_MEASUREMENT_MAX_ATTEMPTS 5
 #define REQUEST_RSSI_MEASUREMENT_DELAY 5000
+#define START_MONITORING_RESPONSE_TIMEOUT_MSEC 3000
+#define START_MONITORING_MAX_ATTEMPTS (60 * START_MONITORING_RESPONSE_TIMEOUT_MSEC) // 3 minutes
```

Comment:
I think that 3 minutes is too long... 30 seconds should be enough.

Reply:
The maximum waiting for a single response is 3 seconds. 3 minutes is the maximum timeout for repeated requests which is a reasonable time for the wifi to be up.
I think I'll keep it.